model/common_structures.smithy

```diff
@@ -59,3 +59,8 @@ structure Hits{
     _source: Document,
     fields: Document
 }
+
+structure Response{
+    status: String
+    message: String
+}
```

Comment:
A structure named response is very vague. Any Response body can be Response

Reply:
> A structure named response is very vague. Any Response body can be Response

Can we name it "UpdateResponse"?

Reply:
because we use it in many API's in update/delete parts